Resources/Prototypes/Nyanotrasen/GameRules/events.yml

```diff
@@ -88,8 +88,8 @@
   noSpawn: true
   components:
     - type: GlimmerEvent
-      minimumGlimmer: 200
-      maximumGlimmer: 500
+      minimumGlimmer: 500
```

Comment:
again, depends on convention but this may need Delta-V comments as this is modifying Nyano code..

Reply:
Updated! Thanks.